libsolidity/interface/CompilerStack.cpp

```diff
@@ -653,6 +664,12 @@ void CompilerStack::compileContract(
 		// CBOR-encoding of {"bzzr0": dev::swarmHash(metadata)}
 		bytes{0xa1, 0x65, 'b', 'z', 'z', 'r', '0', 0x58, 0x20} +
 		dev::swarmHash(metadata).asBytes();
+	if (!safeExperimentalFeature(_contract.sourceUnit().annotation().experimentalFeatures))
+		cborEncodedMetadata =
+		// CBOR-encoding of {"bzzr0": dev::swarmHash(metadata), "experimental": true}
+		bytes{0xa2, 0x65, 'b', 'z', 'z', 'r', '0', 0x58, 0x20} +
```

Comment:
Yes and no. `0xa1` means hash map with one element, `0x65 b z ...` is the key and the hash is the value.

I could pull out the key+value into a separate bytes variable, which is fine for this piece of code, but won't make it valid CBOR elements :)